plugins/experimental/ts_lua/ts_lua_fetch.c

```diff
@@ -260,9 +260,12 @@ ts_lua_fetch_one_item(lua_State *L, const char *url, size_t url_len, ts_lua_fetc
     lua_gettable(L, -2);
 
     if (lua_isstring(L, -1)) {
-      addr = luaL_checklstring(L, -1, &addr_len);
-      rc   = sscanf(addr, "%15s:%d", ipstr, &port);
```

Comment:
rc is a variable used in code block below. So if rc is not 2, then a default will be used.
therefore default is used all the time after this change.